restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/SmsInterpreter.java

```diff
@@ -576,6 +576,7 @@ public void execute(final Object message) throws Exception {
             final SmsMessage record = builder.build();
             final SmsMessagesDao messages = storage.getSmsMessagesDao();
             messages.addSmsMessage(record);
+            getContext().system().eventStream().publish(record);
```

Comment:
i could not find any other occurence of eventStream usage. how is this supposed to be consumed? where is the subcriber regsitered?

Reply:
https://github.com/RestComm/Restcomm-Connect/blob/2d9d4d98a35ef4f5a489cabb93262ba4ba595122/restcomm/restcomm.sdr.api/src/main/java/org/restcomm/connect/sdr/api/SdrService.java#L38